src/Protocol/Protocol_1_9.cpp

```diff
@@ -2665,8 +2665,11 @@ void cProtocol_1_9_0::HandlePacketUseEntity(cByteBuffer & a_ByteBuffer)
 	{
 		case 0:
 		{
-			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand)
-			m_Client->HandleUseEntity(EntityID, false);
+			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand);
+			if (Hand == 0)  // TODO: implement handle off-hand (1) actions
```

Comment:
It currently matches *this* protocol. It may not match a future protocol if Mojang decide to mess with it. I believe it would be better to use a named value that is specific to this protocol - a `static const` value declared either at the top of this function, or the top of this file, so that it's local and constant.